Automattic/jetpack
# Backup: Update references from Jetpack Backup to Jetpack VaultPress Backup

#### Changes proposed in this Pull Request:

* Update Backup plugin file to rename `Jetpack Backup` -> `Jetpack VaultPress Backup`
* Update Backup package UI references from `Jetpack Backup` -> `Jetpack VaultPress Backup`
* Update Backup menu title from `Backup` -> `VaultPress Backup`

#### Screenshots
| Content | Screenshot |
| --- | --- |
| Plugins list page | ![image](https://user-images.githubusercontent.com/1488641/202062038-c7f93c2d-5781-4ffe-a534-dce0b3363564.png) |
| Footer | ![image](https://user-images.githubusercontent.com/1488641/202774066-9148154d-e427-46c7-921e-d798a75ac94b.png) |
| Pricing card | ![image](https://user-images.githubusercontent.com/1488641/202773795-10f56818-2ef5-4245-bb85-716d071f03f0.png) |
| Admin Jetpack Submenu | ![image](https://user-images.githubusercontent.com/1488641/202609400-13d5f6c6-b7d9-48e3-ae9f-ff67f92a1514.png) |
| Page title | ![image](https://user-images.githubusercontent.com/1488641/202609417-b7d090bb-262a-433c-8587-30c127a78a52.png) |  

#### Other information:


#### Jetpack product discussion


* p1HpG7-j2p-p2#comment-58598
* 1202586245449203-as-1203367050452965/f

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Launch a [JN site with Backup standalone and this branch](https://jurassic.ninja/create?jetpack-beta&branches.jetpack-backup=update/vaultpress-backup-rename&wp-debug-log&nojetpack).
* Navigate to `Jetpack` admin menu option.
* You should see:
  * The **pricing card** product name updated from `Jetpack Backup` -> `Jetpack VaultPress Backup`.
  * The **admin Jetpack submenu** from `Backup` -> `VaultPress Backup`.
  * The **page title** in your browser from `Jetpack Backup` -> `Jetpack VaultPress Backup`.
 * Click on `Get Jetpack VaultPress Backup` and purchase a `Jetpack VaultPress Backup` plan.
  * You should see the **footer** changed from `Jetpack Backup` to `Jetpack VaultPress Backup`. 
* Navigate to `Plugins`.
* You should see `Jetpack VaultPress Backup` plugin active instead of `Jetpack Backup` in the **plugins listing page**.
